### PR TITLE
[VA] dotnet 3.0 update

### PR DESCRIPTION
**Pull request by lauren-mills**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/Startup.cs [hunk 1]
```diff
@@ -134,17 +135,24 @@ public void ConfigureServices(IServiceCollection services)
...
+                .UseHttpsRedirection()
                 .UseStaticFiles()
                 .UseWebSockets()
-                .UseMvc();
+                .UseRouting()
+                .UseEndpoints(endpoints =>
+                {
+                    endpoints.MapControllerRoute(
+                        name: "default",
+                        pattern: "{controller=Home}/{action=Index}/{id?}");
```
Comment (xieofxie) [inline templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/Startup.cs]: do we need this?

Comment (lauren-mills) [reply templates/Virtual-Assistant-Template/csharp/Sample/VirtualAssistantSample/Startup.cs]: There may be another way to do it, but I wasn't able to hit the api/messages endpoint without it.

**Comment by lzc850612** [comment]
it looks like nuget restore doesn't support .net core 3.0. can you actually restore the packages locally?

**Comment by ryanisgrig** [comment]
@lzc850612 this may be a solution to nuget restore: https://stackoverflow.com/questions/57501716/pipeline-nuget-restore-failing-on-net-core-3-0-preview-8-project-nu1202

**Comment by lzc850612** [comment]
> @lzc850612 this may be a solution to nuget restore: https://stackoverflow.com/questions/57501716/pipeline-nuget-restore-failing-on-net-core-3-0-preview-8-project-nu1202

i'm not seeing that option in AzureDevOps for dotnet restore. but yeah ideally dotnet restore might do the trick. the other direction is to look to change the pipeline (i don't want us to maintain a customized pipeline though so it has to be something available out of box)

**Review by lzc850612 — APPROVED** [review]

**Review by ryanisgrig — APPROVED** [review]

**Pull request merged**